### DISCUSSION TITLE
uses_non_exempt_encryption false by default?

### DISCUSSION THREAD

**Discussion by Reisclef** [fastlane/fastlane#19796] [Q&A]
Hi,

In the [docs for pilot](https://github.com/fastlane/docs/blob/b52d4c4e1a39c90d0d2136abc7f9a84631f72e95/docs/generated/actions/pilot.md) it states that the default value for `uses_non_exempt_encryption` is `false`.

However, when running the following command from Azure DevOps using Fastlane 2.200.0, we are getting a message in App Store Connect that the app is Missing Compliance:

`fastlane pilot upload --api_key_path /Users/runner/work/_temp/api_key***.json -i /Users/runner/work/r1/a/MyRepo/Apps/MyApp.ipa --skip_submission true --skip_waiting_for_build_processing true -p 12345 --verbose`

![image](https://user-images.githubusercontent.com/9120292/149511205-fae448e3-e03f-466e-ba25-51707c7b306a.png)

I am certain this value should be false based on our usage, and I'd rather do this in CLI rather than in Info.plist.

Am I doing something silly here? Missing something obvious? Are the docs incorrect? I've been looking through the code but haven't found anything obvious.

Any ideas as to why this might be? Do I need to explicitly pass it?

**Comment by mithunproechoes** [comment]
Did you got the answer for this ?

**Comment by Reisclef** [comment]
I did not.

I'm using fastlane with xamarin so we're looking at setting the value within our Xamarin project's .csproj as a temporary workaround.

**Comment by roz0n** [comment]
I am also running into this issue. I think the value is being ignored.

**Comment by Reisclef** [comment]
If someone decides to raise an issue, feel free to quote this issue which I also raised but was auto closed due to lack of activity:

https://github.com/fastlane/fastlane/issues/19991

**Comment by roz0n** [comment]
For the uninitiated: you can manually set [ITSAppUsesNonExemptEncryption](https://developer.apple.com/documentation/bundleresources/information_property_list/itsappusesnonexemptencryption) in `Info.plist` as a workaround

**Comment by MirceaX2Mobile** [comment]
It happens for me too.
